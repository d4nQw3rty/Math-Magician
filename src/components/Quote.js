import React from 'react';

const Quote = () => (
  <>
    <h2>Quote</h2>
    <p
      style={{
        fontSize: '2rem',
        fontWeight: '600',
        marginBottom: '2rem',
        lineHeight: '1em',
        textAlign: 'justyfy',
      }}
    >
      &rdquo;Mathematics is not about numbers, equations, computations, or
      algorithms: it is about understanding&rdquo;
      <span> -- William Paul Thurston</span>
    </p>
  </>
);

export default Quote;
