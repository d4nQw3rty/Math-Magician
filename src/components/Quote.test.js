import React from 'react';
import renderer from 'react-test-renderer';
import Quote from './Quote';

describe('Checking if Quote deploy correctly', () => {
  test('if Calculator renders correctly', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
