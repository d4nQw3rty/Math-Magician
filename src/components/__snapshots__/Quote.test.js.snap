// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checking if Quote deploy correctly if Calculator renders correctly 1`] = `
<p
  style={
    Object {
      "fontSize": "2rem",
      "fontWeight": "600",
      "lineHeight": "1em",
      "margin": "15% 5%",
      "textAlign": "justyfy",
    }
  }
>
  ”Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding”
  <span>
     -- William Paul Thurston
  </span>
</p>
`;
